boards/arm/nrf52_pca20020/doc/nrf52_pca20020.rst

```diff
@@ -0,0 +1,354 @@
++-----+---------------+-----------------------+
+| 3   | SOC           | ANA/DIG0 / P0.02      |
++-----+---------------+-----------------------+
+| 4   | SOC           | ANA/DIG1 / P0.03      |
++-----+---------------+-----------------------+
+| 5   | SOC           | ANA/DIG2 / P0.04      |
++-----+---------------+-----------------------+
+| 6   |               | GND                   |
++-----+---------------+-----------------------+
+| 7   | GPIO Expander | Pin 0                 |
++-----+---------------+-----------------------+
+| 8   | GPIO Expander | Pin 1                 |
++-----+---------------+-----------------------+
+| 9   | GPIO Expander | Pin 2                 |
++-----+---------------+-----------------------+
+| 10  | GPIO Expander | Pin 3                 |
++-----+---------------+-----------------------+
+| 11  | MOSFET 1      | Drain                 |
++-----+---------------+-----------------------+
+| 12  | MOSFET 1      | Source                |
++-----+---------------+-----------------------+
+| 13  | MOSFET 2      | Drain                 |
++-----+---------------+-----------------------+
+| 14  | MOSFET 2      | Source                |
++-----+---------------+-----------------------+
+| 15  | MOSFET 3      | Drain                 |
++-----+---------------+-----------------------+
+| 16  | MOSFET 3      | Source                |
++-----+---------------+-----------------------+
+| 17  | MOSFET 4      | Drain                 |
++-----+---------------+-----------------------+
+| 18  | MOSFET 4      | Source                |
++-----+---------------+-----------------------+
+| 19  |               | VDD                   |
++-----+---------------+-----------------------+
+| 20  |               | GND                   |
++-----+---------------+-----------------------+
+
+MOSFETs
+~~~~~~~
+
+The MOSFETs are attached to the following SOC GPIO pins:
+
++----------+----------+
+| Device   | Gate Pin |
++----------+----------+
+| MOSFET 1 | P0.18    |
++----------+----------+
+| MOSFET 2 | P0.19    |
++----------+----------+
+| MOSFET 3 | P0.20    |
++----------+----------+
+| MOSFET 4 | P0.21    |
++----------+----------+
+
+Power Rails
+-----------
+
+Thing:52 has multiple power rails. The necessary rails for the currently
+supported devices are listed here.
+
++---------+--------------+----------------------+
+| Name    | Derived from | Controlled by        |
++=========+==============+======================+
+| VREG    | The battery  | Always on            |
++---------+--------------+----------------------+
+| VDD_nRF | VREG         | Always on            |
++---------+--------------+----------------------+
+| VDD     | VREG         | SOC pin P0.30        |
++---------+--------------+----------------------+
+| VDD_CCS | VDD          | GPIO expander pin 10 |
++---------+--------------+----------------------+
+
+For a given device to be powered, all of the rails from which device's power
```

Comment:
How about:

    All power rails used for a device must be powered for a given device to be powered.